apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,157 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      this.sinon.useFakeTimers(1);
```

Comment:
It is necessary because by default sinon init its clock to Unix epoch (timestamp of 0) which prevent the function to work as expected.